.github/ISSUE_TEMPLATE/1-roadmap-item.yml

```diff
@@ -0,0 +1,30 @@
+name: '📦 New Roadmap Item'
+labels: 'enhancement'
+description: >
+  This is a project which normally spans several minor releases,
+  and will deliver significant new functionality.
+  Potential examples are "Support for Java 22", "Problems HTML Summary Report", 
+  "Isolated Projects: Incubating Status" or "Declarative Gradle EAP 2".
+  Consider it an "EPIC" in normal Agile terms.
+
+body:
+  - type: textarea
+    attributes:
+      label: Summary / Manifesto
+      description: > 
+        Describe what value this project brings to Gradle users and the ecosystem, and what areas/topics it will address.
```

Comment:
```suggestion
        Describe what value this project brings to Gradle users and the ecosystem, and what areas/topics it will address.
        Make sure to set the context so that it could be understood by the target audience of this project, e.g. a software developer or a build engineer.
```